src/main/java/lithium/university/TwitterPublish.java

```diff
@@ -10,12 +10,15 @@
 public class TwitterPublish {
     /*
     * Takes a message and error checks it before attempting to post to user's twitter
-    * Input: Twitter twitter - twitter instance, String message - message to be posted
+    * Input: Twitter twitter - twitter instance, String message - message to be posted, int tweetTotal - total limited characters
     * Output: true if successful, false otherwise
     * */
     public boolean postToTwitter(Twitter twitter, String message, int tweetTotal) throws TwitterException {
-        if(message.length() > tweetTotal) {
-            System.out.println("Cannot post. Message length should not exceed 280 characters. You're " + (message.length() - tweetTotal) + " characters too long.");
+        if(message.length() <= 0) {
+            System.out.println("Cannot post. Message length needs to be greater than 0.");
+            return false;
+        }else if(message.length() > tweetTotal) {
+            System.out.println("Cannot post. Message length should not exceed " + tweetTotal + " characters. You're " + (message.length() - tweetTotal) + " characters too long.");
```

Comment:
i'd recommend just cannot exceed 280 characters for practicality.  You're less likely to  have errors.  Note: you don't need to actual change it but just an FYI